packages/ckeditor5-dev-utils/tests/translations/translationservice.js

```diff
@@ -17,7 +17,7 @@ const mockery = require( 'mockery' );
 describe( 'translations', () => {
 	describe( 'TranslationService', () => {
 		const sandbox = sinon.sandbox.create();
-		let translationService, stubs;
+		let translationService, TranslationService, stubs;
```

Comment:
`translationService` and `TranslationService` are similar to each other. The first one is an instance of `TranslationService`, the second one is the class definition. In order to avoid troubles, I would like to remove the `translationService` variable from this place and move it to test scenario. Most probably it will be repeated in each test scenario but the whole file will be easier to read.